src/common/gdicmn.cpp

```diff
@@ -455,6 +455,20 @@ wxString wxColourDatabase::FindName(const wxColour& colour) const
     return wxEmptyString;
 }
 
+wxVector<wxString> wxColourDatabase::GetAllNames() const
+{
+   wxColourDatabase * const self = wxConstCast(this, wxColourDatabase);
+   self->Initialize();
+
+   wxVector<wxString> names;
```

Comment:
It won't change the world, but it doesn't cost anything to avoid reallocations by doing
```suggestion
   wxVector<wxString> names;
   names.reserve(m_map->size());
```
neither, so I'd add it here.